src/workerd/util/sqlite.c++

```diff
@@ -51,14 +51,18 @@ namespace workerd {
     int _ec = code; \
     /* SQLITE_MISUSE doesn't put error info on the database object, so check it separately */ \
     KJ_ASSERT(_ec != SQLITE_MISUSE, "SQLite misused: " #code, ##__VA_ARGS__); \
-    SQLITE_REQUIRE(_ec == SQLITE_OK, sqlite3_errmsg(db), " " #code, ##__VA_ARGS__); \
+    int offset = sqlite3_error_offset(db); \
+    bool showOffsetMsg = offset != 1; \
```

Comment:
Should the comparison here be with `-1`? https://sqlite.org/c3ref/errcode.html

Reply:
Yeah, good catch - it was fixed in a commit shortly after opening the PR.

Reply:
My bad, just found that this second :-) Thanks!